### PR TITLE
feat: split out axe tests

### PR DESCRIPTION
**Pull request by devpow112**
https://desire2learn.atlassian.net/browse/QE-264

It's often nicer to have axe tests be able to be run seperatly. It's also easier for test reporting tracking.

### CODE REVIEW

**Comment by bearfriend** [comment]
Not necessarily against this, but did we try/consider using the existing mechanisms to run these separately? 

```
d2l-test-runner -g accessibility
```

I think it still reports tests skipped via grep as failed, but I've got a PR somewhere with that fixed if we want to go this route.

I think there have been arguments for keeping these combined in the past, maybe from @dlockhart ? This way is definitely less prone to issues like false positives, though.

File: src/generators/test-unit-axe/templates/configured/_package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "scripts": {
-    "test": "npm run lint && npm run test:unit",
+    "test": "npm run lint && npm run test:axe && npm run test:unit",
+    "test:axe": "d2l-test-runner axe --chrome",
```
Comment (bearfriend) [inline src/generators/test-unit-axe/templates/configured/_package.json:4]: Any reason we don't want to run them across browsers? I think we've caught some FF-specific issues this way.

Comment (devpow112) [reply src/generators/test-unit-axe/templates/configured/_package.json:4]: I was just switching this to be what we have in a number of other spots
https://github.com/BrightspaceUI/core/blob/main/package.json#L23
https://github.com/BrightspaceUI/htmleditor/blob/main/package.json#L48
I can remove the chrome stuff. It's not a requirement for my stuff

**Comment by devpow112** [comment]
> Not necessarily against this, but did we try/consider using the existing mechanisms to run these separately?
> 
> ```
> d2l-test-runner -g accessibility
> ```
> 
> I think it still reports tests skipped via grep as failed, but I've got a PR somewhere with that fixed if we want to go this route.
> 
> I think there have been arguments for keeping these combined in the past, maybe from @dlockhart ? This way is definitely less prone to issues like false positives, though.

I'm mainly coming at it from a test reporting angle and as it stands right now we couldn't track these correctly without the files split up. Thats why I ended up doing this for `htmleditor` and `labs` when adding test reporting. That may change in the future not sure when/what that would look like exactly.

File: src/generators/test-unit-axe/templates/configured/_package.json (hunk 1 above)
Comment (bearfriend) [reply src/generators/test-unit-axe/templates/configured/_package.json:4]: Well I'll be! Nope, seems like more consideration went into it previusly than I'm giving it now. I'm fine with it matching what we already have.

**Review by bearfriend — APPROVED** [review]

**Pull request merged**